lib/routes/initial_walkthrough/initial_walkthrough.dart

```diff
@@ -146,11 +147,29 @@ class InitialWalkthroughPageState extends State<InitialWalkthroughPage>
         return AlphaWarningDialog();
       },
     );
-    if (approved) _generateMnemonicSeed();
+    if (approved) _startNewNode();
   }
 
-  void _generateMnemonicSeed() {
-    Navigator.of(context).pushNamed("/mnemonics");
+  void _startNewNode() async {
+    final accountBloc = context.read<AccountBloc>();
+    final navigator = Navigator.of(context);
+    var loaderRoute = createLoaderRoute(context);
+    navigator.push(loaderRoute);
+
+    final themeProvider = ThemeProvider.controllerOf(context);
+    try {
+      String mnemonic = bip39.generateMnemonic(strength: 128);
```

Comment:
Why are we not creating the wallet in on the breez-sdk side with e.g rust bitcoin or BDK which is a more reviewed and used ?